pkg/glusterfs/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:7.4.1708
+
+# Copy glusterfs-csi-driver from build directory
+COPY ./build/glusterfs-csi-driver /glusterfs-csi-driver
+
+RUN yum -y install glusterfs-fuse && yum -y install epel-release && yum -y install jq && yum clean all
```

Comment:
All yum things can be one command to ensure fewer fs layers get built.

The `yum clean all` needs a `-y` flag.